corehq/apps/reports/standard/deployments.py

```diff
@@ -318,22 +324,33 @@ def process_rows(self, users, fmt_for_export=False):
                 last_builds = reporting_metadata.get('last_builds')
                 if last_builds:
                     last_build = self.get_data_for_app(last_builds, self.selected_app_id)
+                if device:
+                    device_app_metas = device.get('app_meta')
```

Comment:
Why is this `device_app_metas`, plural? It looks like there's only one meta.

Reply:
renamed